modules/osdk-golang-generate-crd.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-golang-tutorial.adoc
+
+[id="osdk-golang-generate-crd_{context}"]
+= Generating CRD manifests
+
+After the API is defined with `spec` and `status` fields and custom resource definition (CRD) validation markers, you can generate CRD manifests.
+
+.Procedure
+
+* Run the following command to generate and update CRD manifests:
++
+[source,terminal]
+----
+$ make manifests
+----
++
+This Makefile target invokes the `controller-gen` utility to generate the CRD manifests at `config/crd/bases/cache.example.com_memcacheds.yaml`.
+
+[id="osdk-golang-generate-crd-validation_{context}"]
```

Comment:
I'd make this a separate module - it looks like conceptual information, which shouldn't be in a procedure.